test/e2e/network/conntrack.go

```diff
@@ -105,6 +106,12 @@ var _ = SIGDescribe("Conntrack", func() {
 
 		ips := e2enode.CollectAddresses(nodes, v1.NodeInternalIP)
 
+		if framework.TestContext.ClusterIsIPv6() {
```

Comment:
I think that it will be "more solid" if we create the node info independentyl 
https://github.com/kubernetes/kubernetes/blob/ee297b6f4d78daaf2f71c78a05203698581ca40a/test/e2e/framework/node/resource.go#L252

```
                ips := e2enode.GetAddresses(nodes.Items[0], v1.NodeInternalIP)
		clientNodeInfo = nodeInfo{
			name:   nodes.Items[0].Name,
			nodeIP: ips[0],
		}

                ips = e2enode.GetAddresses(nodes.Items[1], v1.NodeInternalIP)
                
		serverNodeInfo = nodeInfo{
			name:   nodes.Items[1].Name,
			nodeIP: ips[0],
		}
```

Reply:
This is much simpler, I will change to this